cmd/snap/cmd_validate.go

```diff
@@ -0,0 +1,191 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/jessevdk/go-flags"
+
+	"github.com/snapcore/snapd/client"
+	"github.com/snapcore/snapd/i18n"
+)
+
+type cmdValidate struct {
+	clientMixin
+	Monitor    bool `long:"monitor"`
+	Enforce    bool `long:"enforce"`
+	Forget     bool `long:"forget"`
+	Positional struct {
+		ValidationSet string `positional-arg-name:"<validation-set>"`
+	} `positional-args:"yes"`
+	colorMixin
+}
+
+var shortValidateHelp = i18n.G("List or set snap validations")
+var longValidateHelp = i18n.G(`
+The validate command lists validation sets or sets validations")
+`)
+
+func init() {
+	addCommand("validate", shortValidateHelp, longValidateHelp, func() flags.Commander { return &cmdValidate{} }, colorDescs.also(map[string]string{
+		// TRANSLATORS: This should not start with a lowercase letter.
+		"monitor": i18n.G("Monitor the given validations set"),
```

Comment:
Is it a single set that is being monitored or multiple sets?

Reply:
Single set can be passed to single invocation of the command, although multiple sets can be monitored/enforced by calling it several times.